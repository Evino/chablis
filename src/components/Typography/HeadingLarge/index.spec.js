import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import HeadingLarge from '.';

const text = 'This is a title';
const element = <HeadingLarge>{text}</HeadingLarge>;
const component = mount(element);

describe('HeadingLarge', () => {
  it('should render a title', () => {
    const tree = renderer
      .create(element)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('should display passed text', () => {
    expect(component.prop('children')).toBe(text);
  });
});
