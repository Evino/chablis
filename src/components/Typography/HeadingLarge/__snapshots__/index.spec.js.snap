// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadingLarge should render a title 1`] = `
<h2
  className="sc-bdVaJa kgXdUJ"
>
  This is a title
</h2>
`;
